app/views/style_guides/index.haml

```diff
@@ -0,0 +1,4 @@
+%h1 Style Guides
+%ul
+  - guide_routes.each do |key, value|
+    %li= link_to(value, "#{key}_style_guide_path")
```

Comment:
one note here.  It might be cleaner to write:
`polymorphic_path([key, :style_guide)` instead of `"#{key}_style_guide_path"`

Reply:
Yes, that seems much better. Thanks for the guidance!